iambic/plugins/v0_1_0/aws/handlers.py

```diff
@@ -396,41 +415,92 @@ async def import_aws_resources(
             )
         )
 
-    if not exe_message.metadata or exe_message.metadata["service"] == "iam":
-        iam_template_map = None
+    tasks += await import_organization_resources(
+        exe_message, config, base_output_dir, messages, remote_worker
+    )
 
-        if not remote_worker or exe_message.metadata:
-            iam_template_map = await get_existing_template_map(
-                repo_dir=base_output_dir,
-                template_type="AWS::IAM.*",
-                nested=True,
-            )
+    # if not exe_message.metadata or exe_message.metadata["service"] == "iam":
+    #     iam_template_map = None
+
+    #     if not remote_worker or exe_message.metadata:
+    #         iam_template_map = await get_existing_template_map(
+    #             repo_dir=base_output_dir,
+    #             template_type="AWS::IAM.*",
+    #             nested=True,
+    #         )
+    #     config.accounts = config.accounts[0:2]
+    #     tasks.append(
+    #         import_service_resources(
+    #             exe_message,
+    #             config,
+    #             base_output_dir,
+    #             "iam",
+    #             [
+    #                 # collect_aws_roles,
+    #                 # collect_aws_users,
+    #                 # collect_aws_groups,
+    #                 # collect_aws_managed_policies,
+    #             ],
+    #             [
+    #                 # generate_aws_role_templates,
+    #                 # generate_aws_user_templates,
+    #                 # generate_aws_group_templates,
+    #                 # generate_aws_managed_policy_templates,
+    #             ],
+    #             messages,
+    #             remote_worker,
+    #             iam_template_map,
+    #         )
+    #     )
 
-        tasks.append(
-            import_service_resources(
-                exe_message,
-                config,
-                base_output_dir,
-                "iam",
-                [
-                    collect_aws_roles,
-                    collect_aws_groups,
-                    collect_aws_users,
-                    collect_aws_managed_policies,
-                ],
-                [
-                    generate_aws_role_templates,
-                    generate_aws_user_templates,
-                    generate_aws_group_templates,
-                    generate_aws_managed_policy_templates,
-                ],
-                messages,
-                remote_worker,
-                iam_template_map,
-            )
+    await asyncio.gather(*tasks)
+
+
+async def import_organization_resources(
+    exe_message: ExecutionMessage,
+    config: AWSConfig,
+    base_output_dir: str,
+    messages: list = None,
+    remote_worker=None,
+) -> list:
+    tasks = []
+    if len(config.organizations) > 0:
```

Comment:
I suggest gating this like so:

```python
if not config.organizations:
   return tasks
# The rest of the code
```

This is a recommended pattern to improve readability, exit early, reduce unnecessary indentation and cognitive burden